projects/ngneat/transloco-locale/src/lib/pipes/transloco-date.pipe.ts

```diff
@@ -0,0 +1,46 @@
+import { Pipe, ChangeDetectorRef, PipeTransform, Inject } from '@angular/core';
+import { isNil } from '@ngneat/transloco';
+import { toDate, localizeDate } from '../helpers';
+import { getDefaultOptions } from '../shared';
+import { LOCALE_CONFIG, LocaleConfig } from '../transloco-locale.config';
+import { TranslocoLocaleService } from '../transloco-locale.service';
+import { DateFormatOptions, Locale } from '../transloco-locale.types';
+import { TranslocoLocalePipe } from './transloco-locale.pipe';
+
+@Pipe({
+  name: 'translocoDate',
+  pure: false
+})
+export class TranslocoDatePipe extends TranslocoLocalePipe implements PipeTransform {
+  constructor(
+    protected translocoLocaleService: TranslocoLocaleService,
+    protected cdr: ChangeDetectorRef,
+    @Inject(LOCALE_CONFIG) private localeConfig: LocaleConfig
+  ) {
+    super(translocoLocaleService, cdr);
+  }
+
+  /**
+   * Transform a date into the locale's date format.
+   *
+   * The date expression: a `Date` object,  a number
+   * (milliseconds since UTC epoch), or an ISO string (https://www.w3.org/TR/NOTE-datetime).
+   *
+   * @example
+   *
+   * date | translocoDate: {} : en-US // 9/10/2019
```

Comment:
`: {}` is redundant.

Reply:
Why not make it part of the options then?

```
  transform(value: Date | string | number, params: DateFormatOptions = {}) {
    if (isNil(value)) return '';

    const {locale = this.getLocale(locale), ...options} = params;

    value = toDate(value);

    return localizeDate(value, locale, {
      ...getDefaultOptions(locale, 'date', this.localeConfig),
      ...options
    });
  }
```